test/e2e/components/vl-multi-select.js

```diff
@@ -1,6 +1,132 @@
-const { VlElement } = require('vl-ui-core').Test;
+const { VlSelect } = require('vl-ui-select').Test;
+const { Pill } = require('./pill');
+const { By } = require('selenium-webdriver');
+
+class VlMultiSelect extends VlSelect {
+
+    async _getRoot() {
+        return this.findElement(By.xpath('../..'));
+    }
+
+    async _getItemList() {
+        const root = await this._getRoot();
+        return root.findElement(By.css('.vl-select__list > div'));
+    }
+
+    async _getItemsToSelect() {
+        const selectList = await this._getItemList();
+        return selectList.findElements(By.css('.vl-select__item'));
+    }
+
+    async _getPills() {
+        const root = await this._getRoot();
+        const pills = await root.findElements(By.css('.vl-pill'));
+        return Promise.all(pills.map(pill => new Pill(this.driver, pill)));
+    }
+
+    async _getPillByValue(value) {
+        const pills = await this._getPills();
+        const elements = await Promise.all(pills.map(async (pill) => {
+            const value = await pill.getValue();
+            return { webElement: pill, value: value };
+        }));
+        return elements.filter(e => e.value == value)[0].webElement;
+    }
+
+    async _getSelectedOptions() {
+        return this.findElements(By.css('option'));
+    }
+
+    async _getInput() {
+        const root = await this._getRoot();
+        return root.findElement(By.css(this.selector + ' ~ input'));
+    }
+
+    async _enterSearchText(searchText) {
+        const input = await this._getInput();
+        return input.sendKeys(searchText);
+    }
+
+    async openDropdown() {
+        const input = await this._getInput();
+        return input.click();
+    }
+    
+    async getSelectedOptionsByValue() {
+        const selectedOptions = await this._getSelectedOptions();
+        return Promise.all(selectedOptions.map(option => option.getAttribute('value')));
+    }
+    
+    async delete(value) {
+        const pill = await this._getPillByValue(value);
+        return pill.remove();
+    }
+    
+    async value(element) {
+        return element.getAttribute('data-value');
+    }
+    
+    async values() {
+        const listItem = await this._getItemsToSelect();
+        return Promise.all(listItem.map(option => this.value(option)));
+    }
+
+    async hasValue(value) {
+        const values = await this.values();
+        return values.includes(value);
+    }
+
+    async hasPartialValue(value) {
+        const values = await this.values();
+        return values.find(v => v.indexOf(value) > -1).length > 0;
```

Comment:
Wat je hier doet is teruggeven of de lengte van de gevonden value > 0.
Als find dan undefined teruggeeft, is er geen length en ga je normaal gezien een fout krijgen.

Ik denk dat je er gewoon het volgende van kunt maken:
```suggestion
        return !!values.find(v => v.indexOf(value) > -1);
```

Dan zet je de return value van find om in een boolean (undefined = false, de rest = true).